README.md

```diff
@@ -1,16 +1,16 @@
-# Unfolded QGIS plugin
+# kepler QGIS plugin by Foursquare
 
-![](https://github.com/UnfoldedInc/qgis-plugin/workflows/Tests/badge.svg)
-[![codecov.io](https://codecov.io/github/UnfoldedInc/qgis-plugin/coverage.svg?branch=main)](https://codecov.io/github/UnfoldedInc/qgis-plugin?branch=main)
-![](https://github.com/UnfoldedInc/qgis-plugin/workflows/Release/badge.svg)
+![](https://github.com/foursquare/qgis-plugin/workflows/Tests/badge.svg)
+[![codecov.io](https://codecov.io/github/foursquare/qgis-plugin/coverage.svg?branch=main)](https://codecov.io/github/foursquare/qgis-plugin?branch=main)
+![](https://github.com/foursquare/qgis-plugin/workflows/Release/badge.svg)
 
 <img src="docs/imgs/foursquare-logo.png" height="100">
 
-This plugin exports [QGIS](http://qgis.org/) vector layers into a format that can be imported into  [Unfolded Studio](https://studio.unfolded.ai/) for further analysis or one-click publishing to the web, after signing up for a free [Unfolded](https://unfolded.ai/) account.
+This plugin exports [QGIS](http://qgis.org/) vector layers into a format that can be imported into  [Foursquare Studio](https://studio.foursquare.com/) for further analysis or one-click publishing to the web, after signing up for a free [Foursquare Studio](https://studio.foursquare.com/) account.
```

Comment:
We need to expand this to talk about both Foursquare and kepler.gl. Probably best to work on this in a Google Doc before a PR, copy the current text into a Google Doc and we can collaborate on that.

Reply:
I've created a "README.md" section within [rebranding RFC doc](https://docs.google.com/document/d/1C0ppvYuK50GBR-ynOlRsGJreBbjWz4KEY0QlQV3bBgk/edit).